uportal-war/src/main/resources/layout/theme/respondr/navigation.xsl

```diff
@@ -105,11 +106,29 @@
     <xsl:param name="CONTEXT"/>  <!-- Catches the context parameter to know how to render the navigation. -->
     <chunk-point/> <!-- Performance Optimization, see ChunkPointPlaceholderEventSource -->
       <nav class="portal-nav">
-        <div class="container-fluid">
-        <a href="#" class="menu-toggle"><i class="fa fa-align-justify"></i> Menu</a>
+        <div id="sidebar">
+          <xsl:choose>
+             <xsl:when test="$USE_OFFCANVAS_MENU='true'">
+                <xsl:attribute name="class">sidebar-offcanvas container-fluid</xsl:attribute>
+              </xsl:when>
+             <xsl:otherwise>
+                <xsl:attribute name="class">container-fluid</xsl:attribute>
+             </xsl:otherwise>
+          </xsl:choose>
```

Comment:
FYI One common approach that is done in other areas is to have something like
https://github.com/Jasig/uPortal/blob/uportal-4.2.1/uportal-war/src/main/resources/layout/theme/respondr/content.xsl#L94-L99
which is used later like at https://github.com/Jasig/uPortal/blob/uportal-4.2.1/uportal-war/src/main/resources/layout/theme/respondr/content.xsl#L119.  This insures there is one place for specifying the CSS classes and HTML markup to reduce the chance someone later on will change one HTML element or attribute but forget to change the other.